README.md

```diff
@@ -857,9 +857,7 @@ This repository also provides one such platforms where contributers come over an
 | [UNO game with computer](https://github.com/kunjgit/GameZone/tree/main/Games/UNO_game_with_Computer) |
 |[Dice_Rolling_Simulator](https://github.com/priyashuu/GameZone/tree/main/Games/Dice_rolling_simulator)|
 |[Space_Dominators](https://github.com/kunjgit/GameZone/tree/main/Games/Space_Dominators)|
-| [Simon_Says](https://github.com/kunjgit/GameZone/tree/main/Games/Simon_Says) |
```

Comment:
these game entry are missing from your file simon says, idle  miner, five nights at freddys
make sure your file is in sync with latest changes